setup.py

```diff
@@ -58,7 +58,7 @@
     long_description=long_description,
     long_description_content_type='text/markdown',
     url="https://github.com/qiskit-community/qiskit-quantinuum-provider",
-    packages=setuptools.find_namespace_packages(include=['qiskit.*']),
+    packages=setuptools.find_namespace_packages(include=['qiskit_quantinuum*']),
```

Comment:
Missed this when we updated the file structure from qiskit/providers/quantinuum to just qiskit_quantiuum.

Tested locally and it allowed me to import the package. 
Let me know if there are any other changes I should make with this.